packages/vscode-extension/package.json

```diff
@@ -103,8 +103,7 @@
         },
         {
           "id": "teamsfx-environment",
-          "name": "Environment",
-          "when": "fx-extension.isMultiEnvEnabled"
```

Comment:
Will this view be displayed when current project is not valid teamsfx project?

Reply:
No, it won't. It will display the welcome view.